app/services/rapid_ready_for_decision/views/shared/intro.erb

```diff
@@ -0,0 +1,20 @@
+<%#
+  Note: This templating language is HTML-inspired. It does not always behave like true HTML.
+  To see limitations, check out the documentation: https://github.com/puzzle/prawn-markup.
+%>
+
+<%= @disability_type.capitalize %> Rapid Ready for Decision | Claim for Increase <br />
+<h1>
+  VHA <%= @disability_type.capitalize %> Data Summary<br/>
```

Comment:
If for some reason there's a more descriptive term we should use, we can use the constants that yoom setup and lookup a proper label.

Reply:
That was the purpose of the `label` key in the Constants hashes.